---
title: "manuscript"
author: "Yuanning Li"
date: "10/17/2018"
output:
  html_document: default
  pdf_document: default
  word_document: default
bibliography: manuscript.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
# Lamellibrachia genome
## Abstract


## Introduction
![](../figures/Picture1.png) 
**Fig 1.** *Lamellibrachia* from seep localities in Gulf of Mexico. 

## Methods

### Biological materials.
Adult *Lamellibrachia* *luymesi* specimens were collected from from seep localities in the Mississippi Canyon at 754 m depth in Gulf of Mexico (N 28°11.58', W 89°47.94'), using the R/V Seward Johnson and Johnson Sea Link in October 2009. All samples were frozen at 80˚C following collection. 

### Genome sequencing and assembly. 
Vestimentum tissue was dissected from one individual of worm, and high molecular weight genomic DNA was extracted using the the DNeasy Blood & Tissue Kit (Qiagen) according to the manufacturer’s protocols. Sequencing a total of six paired-end or mate-pair genomic DNA libraries with insert sizes ranging from 180 bp to 7 kb were performed by by The Genomic Services Lab at the Hudson Alpha Institute in Huntsville, Alabama on an Illumina HiSeq 2000 platform (see details in Table S1). Paired-end libraries (180 bp, 400 bp, 750 bp) were prepared using the 125 bp TrueSeq protocols, and mate-pair libraries (3-5 kbp, 5-7 kbp) were generated using the Illuomina Nextera Mate Pair Library Kit followed by size selection. Moreover, a 10X sequencing library was constructed using the 10X Chromium protocol (10X genomics) at the Hudson Alpha Institute. The finished library were sequenced on an Illumina HiSeqX platform, using paried 151 bp reads with a single 8 bp index read. 

Our workflow of genome assembly was shown in Fig. S1. The paired-end and 10X raw reads were checked using FastQC v0.11.5 [@andrews2010fastqc] and quality filtered (Q score >30) using Trimmomatic v0.36 [@bolger2014trimmomatic]. The estimatation of genome size, level of heterozygosity and repeat contes of the Lamellibrachia genome was determined by analaysing the kmer histograms generated from the paired-end librries using Jellyfish v2.2.3 [@marccais2011fast] and GenomeScope [@vurture2017genomescope] (Fig. S2). The Mate-pair reads were trimmed and sorted using NxTrim v0.3.1 [@o2015nxtrim] which can recgonize and trim the artificial Nextera mate-pair circulation adapters. Only reads from category "mp" (true mate-pair reads) and "unkonwn" (mostly large insert size reads) were used for downstream scaffolding anlaysis. Reads from "pe" (paired-end reads) and "se" (single ends) categories were discarded. 

Given that high heterozygosity of *Lamellibrachia* genome, all reads were assembled using Platanus v1.2.4 [@kajitani2014efficient] with a kmer size of 32. Scaffolding was conducted by mapping Illumina paired-end and mate-pair reads to contigs genrated by Platanus using SSPACE v3.0 [@boetzer2014sspace]. Gaps in the scaffolds were then filled with GapCloser v1.12 [@luo2012soapdenovo2]. Redundant allele scaffods were further remvoed using Redundans v0.13c with default settings [@pryszcz2016redundans]. Genome assembly quality was assessed using QUAST v3.1 [@gurevich2013quast]. Completeness of obtained genome was assessed using BUSCO v3[@waterhouse2017busco] with Metazoa_odb9 database (978 busco genes).

### Transcripome assembly and analysis

### Genome annotation. 
Our genome annotation workflow was shown in Fig. S3. Gene models of *Lamellibrachia* genome were constructed following the Funannotate pipeline 1.3.0 (https://github.com/nextgenusfs/funannotate). Briefly, repeptive regions in the *Lamellibrachia* genome were identified usning RepeatModeler v1.0.8 [@smit2008repeatmodeler] and were subsequently soft-masked using RepeatMasker v4.0.6 [@chen2004using]. RNA-Seq data from different tissue were leveraged to improve the accuracy of gene prediction. RNA-Seq data were assembled *de* *novo* into transcriptomes using Trinity v2.4.0 [@haas2013novo] and HISAT 2.1.0 [@kim2015hisat] was used to algin RNA-Seq reads to the *Lamellibrachia* assembly. Transcrptome assemblies were then passed to PASA pipeline v2.3.3 [@haas2003improving] to identify high quality gene models. The aligned RNA-Seq data wes used to train the *ab* *initio* gene predictions using AUGUSTUS v3.3 [@stanke2006augustus]. Protein alignements from the SwissProt database to "Lamellibrachia" assembly were generated using exonerate [@slater2005automated] and Trinity/PASA transcripts were aligned to the genome using Minimap2 v2.1 [@li2018minimap2]. The tRNA genes were identified using tRNAscan-SE v1.3.1 [@lowe1997trnascan]. Finally, EvidenceModeler 1.1.0 [@haas2008automated] was used to combine all the evidences of gene prediction from protein alignemnts, transcritp alignments, and *ab* *initio* predictions to construct high quality gene models. 
Finally, functional annotations of predicted gene models were analyzed using several curated databases. KEGG orthology was assinged using the KEGG Automatic Annotation server. Gene models were further annotated with domain structure and protein identity by InterProScan [@zdobnov2001interproscan] and SwissProt database, respectively. Secreted proteins were predicted using SignalP [@petersen2011signalp] and Phobius [@kall2007advantages] using InterProScan.

### Molecular clock analysis

### Gene family analysis

### Positive selection analysis


## Results and Discussion

## Conclusion

## Ackowledgements

## Author contribution

\pagebreak

## Supplemental Information


